README.md

```diff
@@ -1,8 +1,162 @@
+                                                                [29] 30  31
+
+            April                           May                            June
+    Su  Mo  Tu  We  Th  Fr  Sa     Su  Mo  Tu  We  Th  Fr  Sa     Su  Mo  Tu  We  Th  Fr  Sa
+                1   2   3 [ 4]                         1 [ 2]         1   2   3   4   5 [ 6]
+    [ 5]  6   7   8   9 [10][11]   [ 3]  4   5   6   7   8 [ 9]   [ 7]  8   9  10  11  12 [13]
+    [12] 13  14  15  16  17 [18]   [10] 11  12  13  14  15 [16]   [14] 15  16  17  18  19 [20]
+    [19] 20  21  22  23  24 [25]   [17] 18  19  20  21  22 [23]   [21] 22  23  24  25  26 [27]
+    [26] 27  28  29  30            [24][25] 26  27  28  29 [30]   [28] 29  30
+                                   [31]
+
+                July                          August                       September
+    Su  Mo  Tu  We  Th  Fr  Sa     Su  Mo  Tu  We  Th  Fr  Sa     Su  Mo  Tu  We  Th  Fr  Sa
+                1   2 [ 3][ 4]                           [ 1]             1   2   3   4 [ 5]
+    [ 5]  6   7   8   9  10 [11]   [ 2]  3   4   5   6   7 [ 8]   [ 6][ 7]  8   9  10  11 [12]
+    [12] 13  14  15  16  17 [18]   [ 9] 10  11  12  13  14 [15]   [13] 14  15  16  17  18 [19]
+    [19] 20  21  22  23  24 [25]   [16] 17  18  19  20  21 [22]   [20] 21  22  23  24  25 [26]
+    [26] 27  28  29  30  31        [23] 24  25  26  27  28 [29]   [27] 28  29  30
+                                   [30] 31
+
+            October                        November                       December
+    Su  Mo  Tu  We  Th  Fr  Sa     Su  Mo  Tu  We  Th  Fr  Sa     Su  Mo  Tu  We  Th  Fr  Sa
+                    1   2 [ 3]                                            1   2   3   4 [ 5]
+    [ 4]  5   6   7   8   9 [10]   [ 1]  2   3   4   5   6 [ 7]   [ 6]  7   8   9  10  11 [12]
+    [11] 12  13  14  15  16 [17]   [ 8]  9  10  11  12  13 [14]   [13] 14  15  16  17  18 [19]
+    [18] 19  20  21  22  23 [24]   [15] 16  17  18  19  20 [21]   [20] 21  22  23  24 [25][26]
+    [25] 26  27  28  29  30 [31]   [22] 23  24  25 [26] 27 [28]   [27] 28  29  30  31
+                                   [29] 30
+
+```bash
+tcal XNYS 1 2020
+```
+
+            January 2020
+    Su  Mo  Tu  We  Th  Fr  Sa
+                [ 1]  2   3 [ 4]
+    [ 5]  6   7   8   9  10 [11]
+    [12] 13  14  15  16  17 [18]
+    [19][20] 21  22  23  24 [25]
+    [26] 27  28  29  30  31
+
+## Frequenty Asked Questions
+
+### Why are open times one minute late?
+
+Due to its historical use in the [Zipline](https://github.com/quantopian/zipline) backtesting system, `trading_calendars` will only indicate a market is open upon the completion of the first minute bar in a day. As an example, on a regular trading day for NYSE:
+
+- 9:30:00 is treated as closed.
+- 9:30:01 is treated as  closed.
+- 9:31:00 is the first time treated as open.
+- 16:00:00 is treated as open
+
+This may change in the future.
+
+
+## Calendar Support
```

Comment:
I purposely moved it lower. I think it is more common for people to want to look up the usage instead of looking up which calendars we support (especially since we have `get_calendar_names` now).